src/features/authPlaygroundAPI/authPlaygroundApiSlice.ts

```diff
@@ -0,0 +1,22 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import { ArkReauthQuery } from "../../lib/authentication/arkBaseQuery";
+import { ExtraType } from "../authentication/authenticationSlice";
+
+const baseurl = "";
+
+export const authPlaygroundApi = createApi({
+  reducerPath: "authPlayground",
+  baseQuery: (args, api, extra) => {
```

Comment:
Move this function to library, ArkReauth

Reply:
sorry i didn't pushed it was there from friday